app/components/search/result-cards/agendaitem.hbs

```diff
@@ -26,12 +26,12 @@
     <AuHeading @level="3" @skin="6" class="au-u-margin-top-tiny">
       <AuLink
         data-test-route-search-result-cards-agendaitem-shorttitle
-        class="au-u-padding-none card-link"
+        class="au-u-padding-none l-result-card__link"
```

Comment:
We rely on this class to avoid double clicks. Will need to be fixed before this can be merged